benchmark/dsymtypes.jl

```diff
@@ -4,32 +4,31 @@ using Random
 using DynamicSumTypes
 
 @kwdef struct A
-    common_field::Int = 1
+    common_field::Int = 0
     a::Bool = true
     b::Int = 10
 end
 @kwdef struct B
-    common_field::Int = 1
-    c::Int = 1
-    d::Float64 = 1.0
-    e::Complex = 1.0 + 1.0im
+    common_field::Int = 0
+    a::Int = 1
+    b::Float64 = 1.0
+    d::Complex = 1 + 1.0im # not isbits
 end
 @kwdef struct C
-    common_field::Int = 1
-    f::Float64 = 2.0
-    g::Bool = false
-    h::Float64 = 3.0
-    i::Complex{Float64} = 1.0 + 2.0im
+    common_field::Int = 0
+    b::Float64 = 2.0
+    d::Bool = false
+    e::Float64 = 3.0
+    k::Complex{Real} = 1 + 2im # not isbits
```

Comment:
oh yeah I realize this field is not aligned somehow, let me also change other packages.